tools/run_tests/run_tests_matrix.py

```diff
@@ -357,7 +357,7 @@ def _create_portability_test_jobs(
         "gcc7",
         # 'gcc10.2_openssl102', // TODO(b/283304471): Enable this later
         "gcc12",
-        # "gcc12_openssl309", // TODO: Enable this later
+        "gcc12_openssl309",
```

Comment:
Have we confirmed that this portability test runs the unit tests for all of the affected files?

Additionally, have we confirmed that this portability test runs at least one of the SSL/TLS e2e tests?

Reply:
We have at least confirmed that it compiles tests - see this comment: https://github.com/grpc/grpc/pull/33944#issuecomment-1661040670

I'm not 100% confident on saying this portability test infra is running all of them, @veblush can you comment?

Reply:
If it's not running the tests, could we use the strategy in https://github.com/grpc/grpc/issues/24960#issuecomment-807158529 to manually run each of the tests to confirm we're not introducing regressions for OpenSSL 3?

Reply:
I've been doing this locally for the tests that I'm changing. When you say "each of the tests" do you mean all of the test targets?